tests/template_tests/syntax_tests/test_querystring.py

```diff
@@ -6,73 +6,81 @@
 
 
 class QueryStringTagTests(SimpleTestCase):
-    def setUp(self):
-        self.request_factory = RequestFactory()
-
-    @setup({"querystring_empty": "{% querystring %}"})
-    def test_querystring_empty(self):
-        request = self.request_factory.get("/")
-        template = self.engine.get_template("querystring_empty")
-        context = RequestContext(request)
-        output = template.render(context)
+
+    request_factory = RequestFactory()
+
+    def assertRenderEqual(self, template_name, request=None, **context):
+        if request is None:
+            request = self.request_factory.get("/")
+        template = self.engine.get_template(template_name)
+        context = RequestContext(request, context)
+        return template.render(context)
```

Comment:
Hi @nessita, I wonder if it makes more sense to actually do the assertion here given the method's name, and similarly to `SafeStringTest`'s `assertRenderEqual()`:
```suggestion
    def assertRenderEqual(self, template_name, expected, request=None, **context):
        if request is None:
            request = self.request_factory.get("/")
        template = self.engine.get_template(template_name)
        context = RequestContext(request, context)
        self.assertEqual(template.render(context), expected)
```

Reply:
@g-nie Thank you for your comment, well spotted!

Originally I left the assertion out because in general I like the assertion to be visible and clear in each test method ("Arrange-Act-Assert"), but you are right that the method name and consistency with test_safestring trumps the `AAA` pattern. Changing and pushing!

Reply:
Thank you @nessita! Looks good to me